docs/tutorials/vagrant-api.rst

```diff
@@ -0,0 +1,195 @@
+Inerfacing with the RESTful API
+===============================
+
+Overview
+--------
+
+In the previous modules, you had the opportunity to experiment with some RackHD APIs. In this module you will learn about two different RESTful endpoints in RackHD and experiment with them.
+
+RackHD is designed to provide a REST (Representational state transfer) architecture to provide a RESTful API. RackHD currently has two RESTful interfaces: a Redfish API and native REST API 2.0.
+
+The Redfish API is compliant with the Redfish specification as an additional REST API. It provides a common data model for representing bare metal hardware, as an aggregate for multiple backend servers and systems.
+
+The REST API 2.0 provides unique features that are not provided in the Redfish API.
+
+Restful API ( v2.0 )
+-----------------
+
+**REST API ( v 2.0) - Get workflow history**
+
+Before the following operations are excuted, you need to ssh the vagrant box first. ``vagrant ssh dev`` can enter the demo environment. ``vagrant ssh dev_ansible`` can enter development environment. The **<Node_ID>** is retrieved from 7.2.
+
+.. code::
+
+  vagrant@rackhd:~$ curl localhost:8080/api/2.0/nodes/<Node-ID>/workflows | jq '.'
```

Comment:
remind user to install jq in the doc.  Besides, introduce"python -mjson" and "jq ''." to users.